tests/modules/misc.c

```diff
@@ -359,6 +359,14 @@ int test_rm_call(RedisModuleCtx *ctx, RedisModuleString **argv, int argc){
     return REDISMODULE_OK;
 }
 
+/* wrapper for RM_Call which also replicates the module command */
+int test_rm_call_replicate(RedisModuleCtx *ctx, RedisModuleString **argv, int argc){
```

Comment:
i now realize that we can use `rm_call_flags` with `!` flag instead of adding this one.

Reply:
That will not work IMO. the '!' flag will only replicate the "called" command and not the module command itself.

Reply:
ohh, right. :facepalm: